test/integration/serializers/sample.json

```diff
@@ -31,20 +34,28 @@
                 ]]
             ]]
           }]
-        ]],
+        ]]],
         ["transition", {}, {"relation": "Frob"}, [
-          ["httpTransaction", {"title": "Get a frob", "description": "Gets information about a single frob instance"}, {}, [
-            ["httpRequest", {}, {"method": "GET"}, null],
+          ["httpTransaction", {"title": "Get a frob"}, {}, [
+            ["copy", {}, {}, "Gets information about a single frob instance"],
+            ["httpRequest", {}, {"method": "GET"}, []],
             ["httpResponse", {}, {"statusCode": 200, "headers": ["httpHeaders", {}, {}, [
-              ["string", {"name": "Content-Type"}, {}, "application/json"],
-              ["string", {"name": "Authorization"}, {}, "Bearer abc123"]
+              ["member", {}, {}, {
+                "key": ["string", {}, {}, "Content-Type"],
+                "value": ["string", {}, {}, "application/json"]
+              }],
+              ["member", {}, {}, {
+                "key": ["string", {}, {}, "Authorization"],
+                "value": ["string", {}, {}, "Bearer abc123"]
+              }]
             ]]}, [
-              ["asset", {"class": ["messageBody"]}, {}, "{\n  \"id\": \"1\",\n  \"tags\": [\n    {\n      \"name\": \"foo\",\n      \"count\": 23\n    }\n  ]\n}\n"],
-              ["asset", {"class": ["messageBodySchema"]}, {}, "{\"type\": \"object\",  \"properties\": {\"id\": {\"type\": \"string\"},\"tags\": {      \"type\": \"array\"}}}"]
+              ["asset", {"classes": ["messageBody"]}, {}, "{\n  \"id\": \"1\",\n  \"tags\": [\n    {\n      \"name\": \"foo\",\n      \"count\": 23\n    }\n  ]\n}\n"],
+              ["asset", {"classes": ["messageBodySchema"]}, {}, "{\"type\": \"object\",  \"properties\": {\"id\": {\"type\": \"string\"},\"tags\": {      \"type\": \"array\"}}}"]
             ]]
           ]]
         ]],
-        ["transition", {"title": "Restart a frob", "description": "Restart a frob instance"}, {"relation": "restart"}, [
+        ["transition", {"title": "Restart a frob"}, {"relation": "restart"}, [
+          ["copy", {}, {}, "Restart a frob instance"],
```

Comment:
I think this is the correct way to go. We need to correct the inconsistency [here](https://github.com/apiaryio/fury.js/pull/45/files#r38299525) and [here](https://github.com/apiaryio/fury.js/pull/45/files#r38299629)